fetch.bs

```diff
@@ -564,6 +572,22 @@ A: 3
  <var>name</var> and <a for=header>value</a> is <var>value</var> to <var>list</var>.
 </ol>
 
+<p>To <dfn export for="header list" id=concept-header-list-set-structured-header>set a structured
+header</dfn> <a for=header>name</a>/<a>structured header</a>
+<var>name</var>/<var>structured header</var> pair in a <a for=/>header list</a> <var>list</var>, run
+these steps:
+
+<ol>
+ <li>Let <var>serialized value</var> be the result of executing the <a>serializing structured
+ headers</a> algorithm on <var>structured header</var>.
+ <li><a for="header list">Set</a> <var>name</var>/<var>serialized value</var> in <var>list</var>.
+</ol>
+
+<p class="note">Structured header values are defined as objects which HTTP can (eventually)
+serialize in interesting and efficient ways. [[!HEADER-STRUCTURE]] For the moment, Fetch only
+supports setting these objects in <a for=/>header lists</a> by first serializing them into a
+string representation.
```

Comment:
I suppose we should nevertheless encourage folks to use this setter to receive benefits later.

Reply:
I added another line about future efficiency gains. Any additional suggestions for the encouragement you'd like to provide here?